OIPA/OIPA/settings.py

```diff
@@ -141,6 +141,7 @@ def rel(*x):
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'admin_reorder.middleware.ModelAdminReorder',
+    'api.middleware.FileExportMiddleware',
```

Comment:
Since you want to merge it into `develop`,  we're gonna need at least a basic test for this (at least for OIPA, not indicators repo) :) At least checking the main logical parts that are important:

1) making sure middleware is only working for `api` Django app 
2) usage of a) specific + b) default  filenames